data/config-schema/src/main/resources/net/corda/schema/configuration/cryptoLibrary/1.0/corda.cryptoLibrary.json

```diff
@@ -5,6 +5,41 @@
   "description": "Configuration schema for the crypto library subsection.",
   "type": "object",
   "properties": {
+    "unmanagedWrappingKeys": {
+      "description" : "Key derivation parameters for unmanaged wrapping keys",
+      "type": "object",
+      "default": {},
+      "patternProperties": {
+        ".*": { 
+          "type" :  "object",
+          "properties": {
+            "salt": {
+              "description": "Salt for the key derivation function",
+              "type": "string"
+            },
+            "passphrase": {
+              "description": "Passphrase for the key derivation function",
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "defaultUnmanagedWrappingKey": {
+      "description": "The default unmanaged wrapping key, which must be in unmanagedWrappingKeys.",
+      "type": "string",
+      "default" : "unmanagedWrappingKey1"
+    },
+    "wrappingKeyRotationPeriod": {
+      "description": "The number of days before a managed wrapping key is considered to have expired.",
+      "type": "integer",
+      "default" : 365
+    },
+    "wrappingKeyRotationWindow": {
+      "description": "The number of days before a deadline when a managed wrapping key rotation should start.",
+      "type": "integer",
+      "default" : 180
+    },
```

Comment:
I think we need to leave these out until we add key rotation (i.e. this should be on a separate branch).
We need some more design around this. I don't know having the 2 is valuable. We just need to know when to start rotating. I think Matthew was also keen that auto rotation takes into account the number of times a key have been used, so let's not settle the config schema for that.
At the moment, the only concern is that we can do a manual rotation of the root keys.

Reply:
yes, config isn't needed for the POC.